### PR TITLE
DDCNLS-1920: Added terms and condition footer for NISP

### PR DESCRIPTION
**Pull request by nehakannaujia**
Changes includes different terms and condition page if parameter is passed and if not it will continue to show by default terms and condition page.

### CODE REVIEW

**Comment by gavinwye** [comment]
@nehakannaujia why does NISP need different terms and conditions?

**Comment by nehakannaujia** [comment]
@gavinwye  : NISP has its own terms and condition page. There are terms and condition based on the service and the content is completely different from the standard t&c.

File: resources/templates/mustache/preprod/govuk-template.mustache.html [hunk 1]
```diff
@@ -241,7 +241,18 @@
                 <ul class="platform-help-links">
                     <li><a href='/help/cookies' target="_blank" data-sso="false" data-journey-click="footer:Click:Cookies">Cookies</a></li>
                     <li><a href='/help/privacy' target="_blank" data-sso="false" data-journey-click="footer:Click:Privacy policy">Privacy policy</a></li>
+
+                    {{#termsAndConditionFooterLinks}}
```
Comment (gordonmcmullan) [inline resources/templates/mustache/preprod/govuk-template.mustache.html]: Do you intend for there to be multiple terms and conditions links? IF not could you rename this to  {{#termsAndConditionsFooterLink}} Plural Conditions and Singular Links. If you do intend to supply multiple t&c links could you just pluralise Conditions.

Comment (nehakannaujia) [reply resources/templates/mustache/preprod/govuk-template.mustache.html]: @gordonmcmullan : Yeah its single link. I will rename it.

**Comment by nehakannaujia** [comment]
@gordonmcmullan : Updated

**Pull request merged**